### DISCUSSION TITLE
Integration for linux

### DISCUSSION THREAD

**Discussion by grambas** [cgiesche/streamdeck-homeassistant#200] [Feature Requests]
Any future plans to write streamdeck-homeassistant for linux users?

**Comment by cgiesche** [comment]
Hi,

the plugin would work under linux. The problem is, that there is no (official) StreamDeck app for linux.

I tried to make https://github.com/timothycrosley/streamdeck-ui work with StreamDeck plugins and so far the results were promising. Unfortunately, my python skills are ... limited (close to not existing) and so i got stuck and never looked back.

I would love to use my stream-deck with linux and don't understand why elgato does not provide their stream-deck app for linux users.

With my limited time, it is already hard to maintain only the plugin. My hope is that Elgato releases the app for Linux, or someone expands the streamdeck-ui project to support official plugins.

**Reply by cgiesche** [reply]
Update: https://github.com/ninjadev64/OpenDeck This sounds promising. Seems to support plugins made for the StreamDeck.
